smrf/data/load_data.py

```diff
@@ -0,0 +1,184 @@
+import logging
+
+import numpy as np
+import utm
+
+from smrf.data import InputCSV, InputGribHRRR, InputNetcdf, InputWRF
+
+
+class InputData():
+    """
+    Class for loading and storing the data, either from
+    - CSV file
+    - HRRR grib files
+    - WRF
+    - Generic gridded NetCDF
+
+
+    The data will be loaded into a Pandas dataframe
+
+    """
+
+    VARIABLES = [
+        'air_temp',
+        'vapor_pressure',
+        'precip',
+        'wind_speed',
+        'wind_direction',
+        'cloud_factor',
+        'thermal',
+        'metadata'
+    ]
+
+    # Data variables and which module they belong to
+    MODULE_VARIABLES = {
+        'air_temp': 'air_temp',
+        'vapor_pressure': 'vapor_pressure',
+        'precip': 'precipitation',
+        'wind_speed': 'wind',
+        'wind_direction': 'wind',
+        'cloud_factor': 'cloud_factor',
+        'thermal': 'thermal'
+    }
+
+    DATA_FUNCTIONS = {
+        'csv': InputCSV,
```

Comment:
I guess you could use the data type now - ` InputCSV.DATA_TYPE`

Reply:
Yep, updated these as well.